lib/apivore.rb

```diff
@@ -2,6 +2,7 @@
 require 'apivore/rspec_helpers'
 require 'apivore/swagger_checker'
 require 'apivore/swagger'
+require 'rspec/rails'
```

Comment:
Does this need to be `rspec/rails`? Also do we need to include rspec in the runtime dependencies now?